test/src/main/scala/scala/test/scala_import/ScalaImportExposesFileJarsTest.scala

```diff
@@ -0,0 +1,23 @@
+package scala.test.scala_import
+
+import com.google.common.cache.Cache
+import org.apache.commons.lang3.ArrayUtils
+import org.specs2.mutable.SpecificationWithJUnit
+import com.lucidchart.relate.SqlRow
+
+class ScalaImportExposesJarsTest extends SpecificationWithJUnit {
```

Comment:
@jjudd are you sure this runs/compiles? I don’t understand what target it belongs to?

Reply:
Nope. My bad. I copied it, but then decided I wanted to add it to the original test instead. I accidentally added it to this new file. I'll make a PR real quick.

Reply:
I have a commit that fixes this and am running tests locally to make sure things are good. I'll open a PR as soon as it is done. Sorry about that.